src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -73,218 +73,209 @@ const LINKS = {
       >
-        <Button.Small.BlackOutline>Check my price</Button.Small.BlackOutline>
-      </a>
-    )
+        {link.title}
+      </NavLink>
+    </div>
+  )
 
-    const { showMobileMenu } = this.state
-    const renderTextLink = (link) => (
-      <div key={link.id} className={styles.textLink}>
-        <NavLink
-          href={link.href}
-          LinkComponent={link.href !== '/login/' ? LinkComponent : null}
-        >
-          {link.title}
-        </NavLink>
-      </div>
+  // These are bespoke icons but design may replace w/FA at a later date
+  const renderSearchIcon = (link) => {
+    const searchIcon = (
+      <svg
+        width="23"
+        height="22"
+        viewBox="0 0 23 22"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <circle
+          cx="8.94497"
+          cy="8.94399"
+          r="7.86196"
+          stroke="#272727"
+          strokeWidth="1.5"
+        />
+        <path
+          d="M14.3063 14.3041L21.6667 20.5876"
+          stroke="#272727"
+          strokeWidth="1.5"
+        />
+      </svg>
     )
 
-    // These are bespoke icons but design may replace w/FA at a later date
-    const renderSearchIcon = (link) => {
-      const searchIcon = (
-        <svg
-          width="23"
-          height="22"
-          viewBox="0 0 23 22"
-          fill="none"
-          xmlns="http://www.w3.org/2000/svg"
-        >
-          <circle
-            cx="8.94497"
-            cy="8.94399"
-            r="7.86196"
-            stroke="#272727"
-            strokeWidth="1.5"
-          />
-          <path
-            d="M14.3063 14.3041L21.6667 20.5876"
-            stroke="#272727"
-            strokeWidth="1.5"
-          />
-        </svg>
-      )
-
-      return (
-        <NavLink
-          className={styles.searchIcon}
-          key={link.id}
-          href={link.href}
-          LinkComponent={link.href !== '/login/' ? LinkComponent : null}
-        >
-          {searchIcon}
-        </NavLink>
-      )
-    }
-
-    const renderAccountIcon = (link) => {
-      const accountIcon = (
-        <svg
-          width="25"
-          height="23"
-          viewBox="0 0 25 23"
-          fill="none"
-          xmlns="http://www.w3.org/2000/svg"
-        >
-          <circle
-            cx="12.25"
-            cy="5.25"
-            r="4.5"
-            stroke="#272727"
-            strokeWidth="1.5"
-          />
-          <path
-            d="M23.5639 22.25H0.751853C0.767099 19.9125 0.905198 17.7185 2.12836 16.0198C3.43803 14.201 6.17055 12.75 12.1579 12.75C18.1452 12.75 20.8778 14.201 22.1874 16.0198C23.4106 17.7185 23.5487 19.9125 23.5639 22.25Z"
-            stroke="#272727"
-            strokeWidth="1.5"
-          />
-        </svg>
-      )
+    return (
+      <NavLink
+        className={styles.searchIcon}
+        key={link.id}
+        href={link.href}
+        LinkComponent={link.href !== '/login/' ? LinkComponent : null}
```

Comment:
same

Also it seems we have some duplication in these NavLinks no?

Reply:
same: https://github.com/getethos/ethos-design-system/pull/275/commits/bfbb5a3f0520f2f157c895f155212546bf09b3a7